src/Pos/WorkMode.hs

```diff
@@ -72,6 +73,7 @@ modeContext [d|
         !(PeerStateTag  ::: PeerStateCtx Production)
         !(JsonLogConfig ::: JsonLogConfig)
         !(LoggerName    ::: LoggerName)
+        !(RelayContext  ::: RelayContext Production)
```

Comment:
It seems wrong to me. What if we want to use something different from `Production`?
However, as I said, I think that `RelayContext` shouldn't have `m` parameter.